src/components/Table/TableBody.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react'
+
+interface TableBodyProps {
+  className?: string
+  children: ReactNode
+}
+
+export function TableBody (props: TableBodyProps): JSX.Element {
```

Comment:
For simple classes like this, too simple without any functionality, don't create a component for it.

Reply:
This is not applicable anymore.